actions/icon_overlay.rb

```diff
@@ -7,28 +7,17 @@ module SharedValues
       ICON_OVERLAY_APP_VERSION = :ICON_OVERLAY_APP_VERSION
     end
 
-    # To share this integration with the other fastlane users:
-    # - Fork https://github.com/fastlane/fastlane
-    # - Clone the forked repository
-    # - Move this integration into lib/fastlane/actions
-    # - Commit, push and submit the pull request
-
-    class IconOverlayAction < Action
+    class IconOverlayAction < Action # rubocop:disable Metrics/ClassLength
       def self.run(params)
         source_path = ENV['ICON_OVERLAY_SOURCE_PATH']
         assets_path = ENV['ICON_OVERLAY_ASSETS_BUNDLE']
         if source_path.to_s.empty? && assets_path.to_s.empty?
```

Comment:
I take it `puts 'Skipping icon overlay' if source_path.to_s.empty? && assets_path.to_s.empty?` is a bit too long here?

Reply:
As I'm not sure we need to force the return and we're not using anything else from this method

Reply:
I'm a bit confused at how this action is actually working in the first place, where is the actual call to `self.overlay`?

Reply:
Found it, was being a noob